packages/vital-elements/doc/ExtendingAndComposing.md

```diff
@@ -340,95 +348,97 @@ const WithHorizontalOrientation = asDialogToken({
+provides the special `Compose` domain. Unlike those in other domains, the keys inside the `Compose`
+domain do not refer to slots; rather they refer to named Variators.
 
-Let's imagine we want our default dialog to be horizontal with a green border
+Let's imagine we want our default `Dialog` to be horizontal with a green border:
 
 ```ts
 const Default = asDialogToken({
   ...Plain, // The old `Default` token defined above.
   Compose: {
     WithGreenBorder,
-    WithHoriontalOrientation,
+    WithHorizontalOrientation,
   },
+  //...
+});
 ```
-Then `as(Default)` is identical in effect to
-`as(Plain, WithGreenBorder, WithHorizontalOrientation)`, but a downstream consumer can
-easily remove or replace the variators:
+
+Then `as(Default)` is identical in effect to `as(Plain, WithGreenBorder,
+WithHorizontalOrientation)`, but a downstream consumer can easily remove or replace the Variators:
 
 ```ts
 const Custom = asDialogToken({
   ...Default,
   Compose: omit(Default.Compose, 'WithGreenBorder'),
-})
+});
 ```
 
-### Avoid extending with a Variator
+### Avoid Extending with a Variator
 
-You may be tempted to use a Variator to extend a base token, eg:
+You may be tempted to use a Variator to extend a base token, e.g.:
 
 ```ts
 const Green = asDialogToken(Default, WithGreenBorder);
 ```
 
-This should generally be avoided. Remember that Variators are intended
-to encapsulate some bit of styling or functionality which can be
-applied (or removed) as a unit.  When you extend with a Variator, you
-are spreading its effects across the domains and slots of the token
-you are extending, which make sit harder to extricate.
+This should generally be avoided. Remember that Variators are intended to encapsulate some bit of
+styling or functionality that can be applied (or removed) as a unit. When you extend with a
+Variator, you are spreading its effects across the domains and slots of the token you are extending,
+which makes it harder to extricate.
 
-Also, while in many cases (including this example), extending will have the same
-effect as composing, this is not always true. The reason, again, is that the
-order in which HOC's are applied can make a difference.
+Also, while in many cases (including this example), extending will have the same effect as
+composing, this is not always true. The reason, again, is that the order in which HOCs are applied
+can make a difference.
 
 > @TODO Example
 
-### Removing with Composiiton
+### Removing with Composition
 
-The composition pattern is great when you want to add some styling or functionality
-to an existing token, but it's not so effective when you want to *remove* something.
-Variators are usually applied "outside" the tokens they extend, so they 
+The composition pattern is great when you want to add some styling or functionality to an existing
+token, but it's not so effective when you want to _remove_ something. Variators are usually applied
+"outside" the tokens they extend, so they 
```

Comment:
@wodenx This sentence is incomplete. I wasn't able to guess how you intended to finish it, so I left it as is.